src/cinder/PolyLine.cpp

```diff
@@ -107,6 +107,17 @@ bool PolyLine<T>::contains( const Vec2f &pt ) const
 	return (crossings & 1) == 1;
 }
 
+template<typename T>
+double PolyLine<T>::area() const
+{
+	typedef boost::geometry::model::polygon<boost::geometry::model::d2::point_xy<double> > polygon;
+
```

Comment:
I realize it might be good to explicitly check for an empty polyline and just return 0 here.